Jenkinsfile

```diff
@@ -19,9 +20,29 @@ node {
       paramsToUseForLimit: 'whitehall',
       throttleEnabled: true,
       throttleOption: 'category'],
+    [$class: 'ParametersDefinitionProperty',
+      parameterDefinitions: [
+        [$class: 'BooleanParameterDefinition',
+          name: 'IS_SCHEMA_TEST',
+          defaultValue: false,
+          description: 'Identifies whether this build is being triggered to test a change to the content schemas'],
+        [$class: 'StringParameterDefinition',
+          name: 'SCHEMA_BRANCH',
+          defaultValue: DEFAULT_SCHEMA_BRANCH,
+          description: 'The branch of govuk-content-schemas to test against']]
+    ],
   ])
 
   try {
+    govuk.initializeParameters([
+      'IS_SCHEMA_TEST': 'false',
+      'SCHEMA_BRANCH': DEFAULT_SCHEMA_BRANCH,
```

Comment:
You're right, and it's a pain. The problem is that Jenkins doesn't seem to pick up build parameters on the very first build of a pipeline job, so we have to do some extra work to make sure they get set to their defaults the on the first build.

A few of us looked into this in alphagov/finder-frontend#278, but we couldn't spot anything that we were doing wrong so I've raised it as an issue with Jenkins: [JENKINS-40574](https://issues.jenkins-ci.org/browse/JENKINS-40574).